spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +112,25 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
+
+  describe "#update" do
+    it 'updates an employee in BambooHR' do
+      xml = YAML.load_file('spec/fixtures/update_employee_xml.yml')
+      response = File.new('spec/fixtures/update_employee_response.json')
+      details = JSON.parse(File.read('spec/fixtures/update_employee_details.json'))
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.